### PR TITLE
[WIP] Support label_smoothed_cross_entropy

### PR DESCRIPTION
**Pull request by ieBoytsov**
By default there is no implementation of cross entropy with soft labels in Pytorch as discussed in #5168

I found a feature request in pytorch for it but it is still not done. 

https://github.com/pytorch/pytorch/issues/7455

There is also discussion where i found an implemented version of this loss. I checked that it performs the same as nn.CrossEntropyLoss given smoothing=0.0 and accurately smoothes labels given nonzero smoothing. I ported it with small refactoring.

Key improvements that are planned in this PR:

1) Add label smoothing cross entropy loss
2) support and parametrise loss choice in `finetune.py`

### CODE REVIEW

**Comment by ieBoytsov** [comment]
The one thing i dont understand is where to initialise this loss in `finetune.py`.

As i see now there is now no loss instances in `finetune.py`, the loss function comes from given model and initialised during modelling_*.py

**Comment by vblagoje** [comment]
This PR seems outdated. The issue has been resolved with PR #5919
